invenio_communities/templates/semantic-ui/invenio_communities/details/upload-instructions.html

```diff
@@ -0,0 +1,14 @@
+<div class="ui container rel-mt-2">
+    <div class="ui divider" aria-hidden="true"></div>
+    <div class="ui text container rel-mt-2">
+
+    <h2 class="ui small header">Want your upload to appear in this community?</h2>
+      <ul>
+        <!-- TODO: This text content needs to be updated (Copied from Zenodo) -->
```

Comment:
I added it there because the text is not completely up to date (it's copied from Zenodo). But as the workflow is not completely decided yet, I guess we don't know what will be the instructions?

Reply:
could we at least change the "Zenodo" part, that it is mergeable ? Then we can re-iterate on the content